ios-xcframework/bin/get_gutenberg_version

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'json'
+
+# get the path to the parent folder using Dir methods
+package_name = 'package.json'
+path_to_package_json = File.join(File.dirname(File.dirname(File.dirname(File.absolute_path(__FILE__)))), package_name)
```

Comment:
What do think using [`File.expand_path`](https://www.rubydoc.info/stdlib/core/File.expand_path), instead of multiple `dirname` calls?

Reply:
Can you elaborate on this suggestion?

The code uses `File.dirname` to have a platform-independent `..`. That is go up three parent folders.

There's no need for this to be platform-independent to be fair, but I find it a good habit to get into 🤷‍♂️

Reply:
BTW, addressing this is no longer necessary since the automation uses `jq` as of 8779e2e88cf17fef54b45d8a14ecbe95b1f3a167